<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>974</width>
    <height>702</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_26">
    <item>
     <widget class="QLabel" name="title">
      <property name="text">
       <string>Recipe Manager</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="2,2,1">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_29">
        <item>
         <widget class="QLabel" name="add_recipe_title">
          <property name="text">
           <string>Add Recipe</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="1,3">
          <item>
           <widget class="QLabel" name="name_label_add_recipe">
            <property name="text">
             <string>Name : </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="add_name">
            <property name="placeholderText">
             <string>Enter Name</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_13" stretch="1,3">
          <item>
           <widget class="QLabel" name="ingredients_label_add_recipe">
            <property name="text">
             <string>Ingredients : </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="add_ingredients">
            <property name="placeholderText">
             <string>Enter Ingredients seperated by commas</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_14" stretch="1,3">
          <item>
           <widget class="QLabel" name="des_label_add_recipe">
            <property name="text">
             <string>Description :</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="add_des">
            <property name="placeholderText">
             <string>Enter Description</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="0">
          <property name="sizeConstraint">
           <enum>QLayout::SetMinimumSize</enum>
          </property>
          <property name="leftMargin">
           <number>100</number>
          </property>
          <property name="rightMargin">
           <number>100</number>
          </property>
          <item>
           <widget class="QPushButton" name="add_recipebtn">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>ADD</string>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_27">
        <item>
         <widget class="QLabel" name="edit_recipe_title">
          <property name="text">
           <string>Edit Recipe</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,3">
          <item>
           <widget class="QLabel" name="name_label_edit_recipe">
            <property name="text">
             <string>Name : </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="edit_name">
            <property name="placeholderText">
             <string>Enter Name</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,3">
          <item>
           <widget class="QLabel" name="ingredients_label_edit_recipe">
            <property name="text">
             <string>Ingredients : </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="edit_ingredients">
            <property name="placeholderText">
             <string>Enter Ingredients seperated by commas</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,3">
          <item>
           <widget class="QLabel" name="des_label_edit_recipe">
            <property name="text">
             <string>Description :</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="add_des_2">
            <property name="placeholderText">
             <string>Enter Description</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <property name="leftMargin">
           <number>100</number>
          </property>
          <property name="rightMargin">
           <number>100</number>
          </property>
          <item>
           <widget class="QPushButton" name="edit_recipebtn">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>CONFIRM</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_30" stretch="1,1">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_25" stretch="2,1,0,1">
          <item>
           <widget class="QLabel" name="find_recipe_title">
            <property name="text">
             <string>Find Recipe</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="findby">
            <item>
             <property name="text">
              <string>By Name</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>By Ingredients</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="search_value">
            <property name="placeholderText">
             <string>Enter Name</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="find_recipebtn">
            <property name="text">
             <string>FIND</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_23" stretch="2,1,1">
          <item>
           <widget class="QLabel" name="delete_recipe_title">
            <property name="text">
             <string> Delete Recipe</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="name_label_delete_recipe">
              <property name="text">
               <string>Name : </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="delete_name">
              <property name="placeholderText">
               <string>Enter Name</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="delete_recipebtn">
            <property name="text">
             <string>CONFIRM</string>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
            <property name="default">
             <bool>false</bool>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableView" name="tableView"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
